### DISCUSSION TITLE
putting items in the storage in range of crafting table with the mod

### DISCUSSION THREAD

**Discussion by Craft94land** [samolego/ClientStorage#26] [Q&A] [answered]
Everytime i put something in the inventory gui of the crafting table it crashes my game
im using this mod and the iron chests mod and it always crashes my game 

I hope someone can help with it

[crash-2023-03-01_09.52.56-client.txt](https://github.com/samolego/ClientStorage/files/10859085/crash-2023-03-01_09.52.56-client.txt)

**Comment by samolego** [comment]
This should be reported as issue ;)

Could you link me the mod?

**Comment by samolego** [comment, marked as answer]
This is actually the IronChests mod issue, reported it here: https://github.com/4nner/IronChests/issues/38
